airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/JavaBaseConstants.kt

```diff
@@ -3,6 +3,13 @@
  */
 package io.airbyte.cdk.integrations.base
 
+import java.util.*
+import org.apache.commons.lang3.StringUtils
+
+fun upperQuoted(column: String): String {
```

Comment:
Why is this outside of the object, and not used in the object?

Reply:
It's used in the stacked PR on top - it could probably be moved to another package if that makes more sense. 

I thought keeping static methods outside of an object was the _Kotlin way_  - do we prefer a different approach